package.json

```diff
@@ -1,207 +1,212 @@
+        "@loadable/babel-plugin": "^5.10.0",
+        "@testing-library/react": "^9.1.4",
+        "@testing-library/react-hooks": "^2.0.1",
+        "amphtml-validator": "^1.0.23",
+        "babel-eslint": "^10.0.3",
+        "babel-jest": "^24.9.0",
+        "babel-loader": "^8.0.6",
+        "babel-plugin-dynamic-import-node": "^2.3.0",
+        "babel-plugin-styled-components": "^1.10.6",
+        "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+        "bbc-a11y": "^2.3.2",
+        "brotli-webpack-plugin": "^1.1.0",
+        "chalk": "^2.4.2",
+        "chrome-launcher": "^0.11.2",
+        "compression-webpack-plugin": "^3.0.0",
+        "copy-webpack-plugin": "^5.0.4",
+        "cypress": "3.4.1",
+        "enzyme": "^3.10.0",
+        "enzyme-adapter-react-16": "^1.14.0",
+        "enzyme-to-json": "^3.4.0",
+        "eslint": "^6.4.0",
+        "eslint-config-airbnb": "^18.0.1",
+        "eslint-import-resolver-alias": "^1.1.2",
+        "eslint-config-prettier": "^6.3.0",
+        "eslint-plugin-cypress": "^2.6.1",
+        "eslint-plugin-import": "^2.18.2",
+        "eslint-plugin-json": "^1.4.0",
+        "eslint-plugin-jsx-a11y": "^6.2.3",
+        "eslint-plugin-prettier": "^3.1.0",
+        "eslint-plugin-react": "^7.14.3",
+        "eslint-plugin-react-hooks": "^2.0.1",
+        "husky": "^3.0.5",
+        "inspectpack": "^4.2.2",
+        "jest": "^24.9.0",
+        "jest-fetch-mock": "^2.1.2",
+        "jest-styled-components": "^6.3.3",
+        "lint-staged": "^9.2.5",
+        "npm-run-all": "^4.1.5",
+        "prettier": "^1.18.2",
+        "prop-types": "^15.7.2",
+        "react-test-renderer": "^16.9.0",
+        "start-server-webpack-plugin": "^2.2.5",
+        "storybook-chromatic": "^2.2.2",
+        "stylelint": "^10.1.0",
+        "stylelint-config-recommended": "^3.0.0",
+        "stylelint-config-styled-components": "^0.1.1",
+        "stylelint-processor-styled-components": "^1.8.0",
+        "supertest": "^4.0.2",
+        "swagger-cli": "^2.3.1",
+        "terser-webpack-plugin": "^2.1.0",
+        "timemachine": "^0.3.0",
+        "wait-on": "^3.3.0",
+        "webpack": "^4.40.2",
+        "webpack-bundle-analyzer": "^3.5.0",
+        "webpack-cli": "^3.3.8",
+        "webpack-dev-server": "^3.8.1",
+        "webpack-merge": "^4.2.2",
+        "webpack-node-externals": "^1.7.2",
+        "yaml-js": "^0.2.3"
+    },
+    "spec": {
+        "executable": [
+            "./bake-scripts/*"
+        ],
+        "post": [
+            "mkdir -p /etc/bake-scripts/",
+            "cp -R -af /usr/lib/simorgh/bake-scripts/* /etc/bake-scripts"
+        ],
+        "prune": false,
+        "requires": [
+            "td-agent",
+            "cloudteam-td-agent-setup",
+            "dial-agent"
+        ]
+    },
+    "lint-staged": {
+        "*.{js,jsx,json}": "eslint",
+        "src/**/*.{js,jsx}": "stylelint"
+    },
+    "buildMetadata": {
+        "commitHash": "383522b",
```

Comment:
question, what's the significance of the `commitHash` value ??

Reply:
This will be the commit hash that was checkout out when we build and deploy to test and live.

This value can then be used to see exactly what commit is on the environment at a given time.

And can also be used to ensure our e2e pipeline checkouts the same commit that is on the environment it will testing against

Reply:
The hash here is just an example, this data will be replaced by the build pipeline each time we merge to latest